specifyweb/backend/permissions/migrations/0008_attachment_import_role.py

```
@@ -85,6 +85,7 @@ class Migration(migrations.Migration):
     dependencies = [
         ('permissions', '0007_add_stats_edit_permission')
     ]
+    # BUG: Make this reversible
     operations = [
         migrations.RunPython(add_attachment_import_role)
-    ]
\ No newline at end of file
+    ]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)